firestore/ios/firestore-snippets-cpp/AppDelegate.mm

```diff
@@ -17,17 +17,23 @@
 #import "AppDelegate.h"
 #import <Firebase/Firebase.h>
 #include "firebase/app.h"
+#include "firebase/firestore.h"
 
 @interface AppDelegate ()
 
 @end
 
 @implementation AppDelegate
 
-
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [FIRApp configure];
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-variable"
+  // [START get_firestore_instance]
   firebase::App::Create();
+  auto db = firebase::firestore::Firestore::GetInstance();
```

Comment:
Nit: it is recommended in the Google style guide (and is generally a good practice) to always use the most qualified form of `auto` that is applicable. One way to look at it is this: `auto` hides a lot of information (it's almost like a wildcard), so it's useful to strive to keep as much information as possible. In this instance, it should be `auto* db = ...` so that the reader immediately knows it is a pointer.

Reply:
With the comment above I'll just change this to `Firestore *` so the type is explicit in the snippet.